netci.groovy

```diff
@@ -64,7 +64,7 @@ platformList.each { platform ->
         if (os == 'PortableLinux') {
 
             // Trigger a portable Linux build that runs on RHEL7.2
-            buildCommand += " --portableLinux"
+            buildCommand += " --portable"
```

Comment:
This is "-portable" (see https://github.com/dotnet/core-setup/blob/master/build_projects/dotnet-host-build/build.sh#L110).